docs/UserGuide.md

```diff
@@ -1527,7 +1526,7 @@ Format: `addtoconsult INDEX n/STUDENT_NAME …`
 
 #### ❌Removing Students from a Consultation: `removefromconsult`
 
-You can remove your student(s) from a consultation.
+You can remove your student(s) from a consultation if there is a need to reschedule or cancel their participation in the scheduled consultation.
```

Comment:
You can remove your student(s) from a consultation if you need to reschedule or cancel your student's participation in the scheduled consultation.